### PR TITLE
joern-scan: add config for query dump destination

### PR DESCRIPTION
**Pull request by ursachec**

### CODE REVIEW

**Review by mpollmeier — APPROVED** [review]
👍🏻 
if you want to take it one step further: define a static `DefaultDumpDestination` e.g. in `object JoernScan` and refer to it in both the helptext and the parameter

File: joern-cli/src/main/scala/io/joern/joerncli/JoernScan.scala [hunk 1]
```diff
@@ -59,7 +60,11 @@ object JoernScan extends App with BridgeBase {
 
     opt[Unit]("dump")
       .action((_, c) => c.copy(dump = true))
-      .text("Dump available queries to file")
+      .text("Dump available queries to a temporary file")
```
Comment (mpollmeier) [inline joern-cli/src/main/scala/io/joern/joerncli/JoernScan.scala:63]:
```suggestion
      .text("Dump available queries to a file")
```
I'd keep it as before - the file isn't automatically removed, at least not by us...

**Review by mpollmeier — APPROVED** [review]

File: joern-cli/src/main/scala/io/joern/joerncli/JoernScan.scala (hunk 1 above)
Comment (ursachec) [reply joern-cli/src/main/scala/io/joern/joerncli/JoernScan.scala]: :+1: good point, thx

**Comment by ursachec** [comment]
> 👍🏻 if you want to take it one step further: define a static `DefaultDumpDestination` e.g. in `object JoernScan` and refer to it in both the helptext and the parameter

yess, much nicer, danke

**Pull request merged**